tests/translator/output/error_schema_validation_wrong_type.json

```
@@ -0,0 +1,8 @@
+{
+  "errorMessage": "Invalid Serverless Application Specification document. Number of errors found: 1. Resource with id [MyConnector] is invalid. 'Id' is missing or not a string.",
+  "errors": [
+    {
+      "errorMessage": "Resource with id [MyConnector] is invalid. 'Id' is missing or not a string."
+    }
+  ]
```

I think the `errors` is unused?

```suggestion
  "errorMessage": "Invalid Serverless Application Specification document. Number of errors found: 1. Resource with id [MyConnector] is invalid. 'Id' is missing or not a string."
```